includes/specials/SpecialSearch.php

```diff
@@ -439,39 +413,42 @@ public function showResults( $term ) {
 
 	/**
 	 * @param $t Title
+	 * @return void
+	 * @throws FatalError
+	 * @throws MWException
 	 */
 	protected function showCreateLink( $t ) {
-		global $wgOut;
-
 		/* Wikia change begin - @author: Macbre */
 		/* Don't show "create an article" link in Oasis */
 		if ( F::app()->checkSkin( [ 'oasis' ] ) ) {
-			return '';
+			return;
```

Comment:
Return value is never used - it's either void or this `''`, lets make it always void.